data/map planets.txt

```diff
@@ -277,6 +285,15 @@ planet Baianus
 	description `	Unlike Titan, however, Baianus is regularly wracked by both waves of heat and energy, and a stay of any length on the surface is likely to see waves of auroras accompanied by glows and sparks rippling across the terrain. Your environmental system strongly recommends remaining inside your ship whenever ion storms are active.`
 	government Uninhabited
 
+planet Bailey
+	attributes CCOR mining
+	landscape land/lava3
+	description `Bailey is a dead, nearly tidally locked world scarcely larger and more habitable than Mercury. Its cool side has hardened into a mineral-rich crust, a rarity in this region of space. Despite these conditions, Bailey's bountiful crust has brought several mining expeditions to the planet.`
+	description `	Like aphids sucking the sap from a plant, semi-mobile mining stations strip and refine the minerals, a method originally developed by the Confederated Councils. Since their fall, the pirates of the region have maintained the stations - although with considerably less organization than what was upheld by the central government.`
+	spaceport `Given the roving nature of the settlements, and the lack of a livable atmosphere, the spaceport and other major infrastructure are located in orbit. Transit to and from the planet is facilitated by a clever network of skyhooks, which utilize fewer amounts of rare and expensive resources compared to the more common repulsor technology used elsewhere.`
```

Comment:
1. The planet sprite used is just a normal lava planet that wouldn't suggest any sort of in-orbit landing zone.
2. This implies that repulsor tech is in some way expensive or difficult to get your hands on, yet repulsor tech is almost as old as the hyperdrive and is actually in all likelihood very cheap/available.